ipfs/roles/installations/tasks/main.yml

```
@@ -0,0 +1,56 @@
+    dest: /tmp/ipfs.linux-amd64.tar.gz
+  become: true
+
+- name: Extract and move go-ipfs to a new location
+  unarchive:
+    src: /tmp/ipfs.linux-amd64.tar.gz
+    dest: "{{ user_home }}/"
+    remote_src: yes
+
+- name: Move the ipfs binary to its respective folder
+  shell: mv "{{ user_home }}/go-ipfs/ipfs" /usr/local/bin
+  become: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)